### PR TITLE
Adjust Info Screen To Announce New Release Features 2.20 (EXPOSUREAPP-12082)

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
Screenshot 2022-03-15 at 15 43 03

### CODE REVIEW

**Comment by Ein-Tim** [comment]
> Anzeige erhöhten Risikos verkürzt
Ein erhöhtes Risiko nach einer Risikobegegnung wird nur noch für 10 anstatt für 14 Tage angezeigt.

This is somehow very short, I think. It is not only increased risk, but also low risk. It is everything, the app only considers 10 days. Or am I mistaken?

**Review by CV113 — APPROVED** [review]
UA approved

**Comment by KathrinLuecke** [comment]
@Ein-Tim Thanks, yes it might be more correct to mention more generally that the time frame for the risk calculation has been reduce to 10 days.  I'll check that with the colleagues.

**Review by mtwalli — APPROVED** [review]

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml [hunk 1]
```diff
@@ -17,30 +17,38 @@
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
-        <item>Registrierung von PCR-Schnelltests</item>
-        <item>Zuordnung von Zertifikaten </item>
-        <item>Zertifikate aktualisieren</item>
+        <item>Anzeige erhöhten Risikos verkürzt</item>
+        <item>Risikoanzeige nach positivem Testergebnis</item>
+        <item>Nachricht zu neuer Risikobegegnung</item>
```
Comment (vaubaehn) [inline Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:22]: The headline is wrong and does not fit to the description. Please verify.

**Review by vaubaehn — CHANGES_REQUESTED** [review]
One problem identified: One headline is not fitting to text.

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml (hunk 1 above)
Comment (vaubaehn) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:22]: In iOS it's "Nachricht über Status-Änderung": https://github.com/corona-warn-app/cwa-app-ios/blob/848bc38b7bfa9906e2c55e9421d2b7ec49f9b599/src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings#L1667

Comment (mtwalli) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:22]: @KathrinLuecke , @CV113  FYI

Comment (KathrinLuecke) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:22]: There should be both headlines
item 3 : "Nachricht zu neuer Risikobegegnung" and item 4: "Nachricht über Statusänderung"
I see however, that item 4 is missing in Figma. I will add it asap.

**Comment by KathrinLuecke** [comment]
@Ein-Tim We decided to keep point 1 "Anzeige erhöhten Risikos verkürzt" as this is what the user will notice / see in the app, wheras the reduced time frame for the risk calculation as such is not visible. It's just the technical think in the background.

**Comment by Ein-Tim** [comment]
@KathrinLuecke 

But it affects also how long exposures with low risk are shown, or not?

**Comment by Ein-Tim** [comment]
@KathrinLuecke 

Okay, I don't think that this is optimal but it's ok.
I guess the blog post will cover this more detailed.

Thanks for considering my feedback!

**Comment by vaubaehn** [comment]
Hi @jurajkusnier ,
https://github.com/corona-warn-app/cwa-app-android/pull/4932#pullrequestreview-910396400 had not been solved yet. Will there be a follow-up PR?

**Comment by jurajkusnier** [comment]
> Hi @jurajkusnier ,
> [#4932 (review)](https://github.com/corona-warn-app/cwa-app-android/pull/4932#pullrequestreview-910396400) had not been solved yet. Will there be a follow-up PR?

Hi @vaubaehn sorry, I didn't mention it. I closed this PR as there will be follow-up PR from @AlexanderAlferov

**Comment by AlexanderAlferov** [comment]
@vaubaehn new PR was created: https://github.com/corona-warn-app/cwa-app-android/pull/4941 👀

**Pull request merged**